netflix/src/app/(commonLayout)/details/[id]/page.tsx

```diff
@@ -0,0 +1,30 @@
+import DetailTop from '@/components/details/DetailTop';
+import DetailBottom from '@/components/details/DetailBottom';
+import PlayBtn from '@/components/common/PlayBtn';
+import FooterNav from '@/components/common/FooterNav';
+import { getMovieDetails } from '@/api/getMovies';
+import { MovieTypes } from '@/types/Movie';
+
+interface DetailPageProps {
+  params: { id: string };
+}
+
+const Page = async ({ params }: DetailPageProps) => {
+  const movie: MovieTypes = await getMovieDetails(params.id);
+
+  return (
+    <div>
+      <div className="flex flex-col w-full">
+        <DetailTop poster_path={movie.poster_path} />
+        <div className="pt-3.5 pl-8 pr-8">
```

Comment:
```suggestion
        <div className="pt-3.5 pl-8 pr-8">
```
 <div className="pt-3.5 px-8 flex flex-col items-center">

 여기에서는 이렇게 고쳐주면 가운데로 정렬돼요! 
그런데 이렇게 바꾸면 밑에 overview 부분도 조정해야 될 것 같아요 😿